web_console/web_terminal/troubleshooting-web-terminal.adoc

```diff
@@ -9,6 +9,44 @@ toc::[]
 [id="troubleshooting-web-terminal-network-policies"]
 == Web terminal and network policies
 
-The web terminal might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state.
+The web terminal might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state until a context deadline exceeded error occurs.
 
-To avoid this issue, ensure that the network policies for namespaces that are used for terminals allow ingress from the `openshift-console` and `openshift-operators` namespaces.
\ No newline at end of file
+To avoid this issue, ensure that the network policies for namespaces that are used for terminals allow ingress from the `openshift-console` and `openshift-operators` namespaces.
+
+The following samples show `NetworkPolicy` objects for allowing ingress from the `openshift-console` and `openshift-operators` namespaces.
+
+.Example network policy for allowing ingress from the `openshift-console` namespace
```

Comment:
Can these headings be shortened? Standard headings are very brief ("Example input", etc.).

Reply:
Updated to:
```
Allowing ingress from the `openshift-console` namespace
```

WDYT?